components/engagement/EngSelectContacts.vue

```diff
@@ -133,13 +173,23 @@ td {
   background-repeat: no-repeat;
   @apply mt-6 pl-6 pr-4 font-bold;
 }
-
 button {
   @apply font-bold text-lg
 }
-
 button:focus,
 option:focus {
   @apply outline-none;
 }
 </style>
+© 2020 GitHub, Inc.
```

Comment:
I noticed these Github lines in a couple of files.  might want to remove those there Bud...  besides that it tests well

Reply:
Thanks for point it out, it's fixed